src/legacy/server/saved_objects/migrations/core/elastic_index.test.ts

```diff
@@ -340,7 +341,11 @@ describe('ElasticIndex', () => {
   describe('write', () => {
     test('writes documents in bulk to the index', async () => {
       const index = '.myalias';
-      const callCluster = sinon.stub();
+      const callCluster = jest.fn(() =>
+        Promise.resolve({
+          items: [],
+        })
+      );
```

Comment:
nit: There's a shorter alternative you can do throughout, `jest.fn().mockResolvedValue({ items: [] })`